docs/index.md

```diff
@@ -3,26 +3,26 @@ layout: page
 title: Documentation
 ---
 
-[Fluo] and [Fluo Recipes] have seperate documentation as they are different repositories with their own release cycle.
+[Apache Fluo] and [Apache Fluo Recipes] have seperate documentation as they are different repositories with their own release cycle.
 
-## Fluo
+## Apache Fluo
 
 Latest release is `{{ site.latest_fluo_release }}` made on {{ site.latest_fluo_release_date }}.
```

Comment:
Good point.  Also, this should be noted in archives of release notes, release summaries, and API.  I will work on this and push another commit.